source/intercept/CMakeLists.txt

```diff
@@ -28,6 +28,20 @@ target_link_libraries(events_db_a PUBLIC
         fmt::fmt)
 
 
+# Intercept json configuration code
+add_library(intercept_json_a OBJECT)
+target_sources(intercept_json_a
+        PRIVATE
+            source/Configuration.cc
+        INTERFACE
+            $<TARGET_OBJECTS:intercept_json_a>
+        )
+target_link_libraries(intercept_json_a PUBLIC
+        result_a
+        fmt::fmt
+        nlohmann_json::nlohmann_json)
+target_compile_options(intercept_json_a PRIVATE -fexceptions)
```

Comment:
The configuration code for intercept requires a different compilation target because nlohmann json requires exceptions.